config/database.js

```diff
@@ -0,0 +1,58 @@
+const Sequelize = require('sequelize');
+const path = require('path');
+
+const connection = require('./connection');
+
+let database;
+
+switch (process.env.NODE_ENV) {
+  case 'production':
+    database = new Sequelize(
```

Comment:
All cases are the same: just create an `envConnection` that is `connection.production`, `connection.development` or whatever according to the environment.